weaviate/collections/aggregations/over_all.py

```diff
@@ -46,14 +47,33 @@ async def over_all(
             if (return_metrics is None or isinstance(return_metrics, list))
             else [return_metrics]
         )
-        builder = self._base(return_metrics, filters, total_count)
-        builder = self._add_groupby_to_builder(builder, group_by)
-        res = await self._do(builder)
-        return (
-            self._to_aggregate_result(res, return_metrics)
-            if group_by is None
-            else self._to_group_by_result(res, return_metrics)
-        )
+        if isinstance(group_by, str):
+            group_by = GroupByAggregate(prop=group_by)
+
+        if self._connection._weaviate_version.is_lower_than(1, 28, 4):
```

Comment:
should we do 1.29.0?

Reply:
The aggregate protos and grpc implementation are on `stable/v1.28` of core so will be available in `1.28.4`

Reply:
it is a rather big change for a point release :)

Reply:
Hmm, that is true. It is in `stable/v1.28` now so it'll unavoidably go into `1.28.4`, unless we revert. Then, from the client's PoV, it's just an implementation detail of the client/server relationship; there are no breaking changes

Reply:
that's why I'd suggest that we only switch over to GRPC for 1.29.0 onwards. Doesn't matter that aggregate GRPC is available for ealier versions if we don't use it